src/less/actionable/base/actionable.less

```diff
@@ -25,6 +25,7 @@ a.icon-link {
     svg {
         color: @actionable-icon-color-active;
         height: 100%; // vertical centering
+        max-width: 100%; // helps when width & height HTML attrs not set
```

Comment:
Doesn't each icon have a `width` set for their `icon--${name}` css class? So do we not need this if we correctly set the right class?

Reply:
Without it, we see issues in custom infotip in coreui, even when the svg has a css width:

![Screen Shot 2019-11-19 at 1 41 18 PM](https://user-images.githubusercontent.com/38065/69188954-601acb00-0ad2-11ea-88c5-fb5dc7a68538.png)

![Screen Shot 2019-11-19 at 1 43 02 PM](https://user-images.githubusercontent.com/38065/69189057-8e000f80-0ad2-11ea-9f6b-c32708295bb3.png)

Applying the max-width fixes this issue:

![Screen Shot 2019-11-19 at 1 43 48 PM](https://user-images.githubusercontent.com/38065/69189164-b38d1900-0ad2-11ea-8d82-88437f46656e.png)

![Screen Shot 2019-11-19 at 1 43 53 PM](https://user-images.githubusercontent.com/38065/69189175-b720a000-0ad2-11ea-866c-8371fbc151eb.png)

I think if this works for now, let's keep it, but would like to spend more time investigating what's going on in future. Maybe as part of the whole icon refactor in v10.